tabled/examples/derive/crate.rs

```diff
@@ -0,0 +1,39 @@
+//! This example demonstrates using the [attribute macro](https://doc.rust-lang.org/reference/procedural-macros.html#attribute-macros)
+//! [`skip`] to omit specific fields from becoming columns in a [`Table`] display.
+//!
+//! * Note how [`skip`] annoys [clippy](https://doc.rust-lang.org/clippy/) with `dead_code`
+//! warnings. This can be addressed with compiler overrides like `#[allow(dead_code)]`.
+
+pub mod unknown_crate {
+    pub use ::tabled::{self, Table, Tabled};
+}
```

Comment:
Yes, but it's correct.
Cause it's expanded down to `unknown_crate::Tabled`

PS: Sorry for late answer.
PS2: But you're right that it can be deleted to not be misleading